NSS-Day-Cohort-42/rare-media-moguls
# login functions as expected aside from bug spelled out in ticket #100

#### Changes Made
1. Modified file `Login.js` to check email and password input
2. Modified file `Login.js` to use props
3. Modified `Rare.js` to work in Login routing and props
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-login-auth
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When a non-authed user accesses the app, they will see a login page
> When a user enters an email and password that match an entry in the db, they will be redirected to the home page
> When a user enters an email that is correct and a password that is incorrect, an error modal appears
> When a user enters an incorrect email, nothing occurs (see bug ticket #100)
